src/electron/routing_service.ts

```diff
@@ -58,7 +58,7 @@ interface NetError extends Error {
 
 // Abstracts IPC with OutlineService in order to configure routing.
 export class RoutingService {
-  private ipcConnection: net.Socket;
+  private ipcConnection!: net.Socket;
```

Comment:
Maybe `net.Socket | undefined`, since it's not initialized in the constructor?

Reply:
Yep, I just made a change throughout to avoid the "definite assignment assertion operator".